mozci/task.py

```diff
@@ -614,6 +647,31 @@ def failing_tasks(self):
             )
         ]
 
+    def is_confirmed_failure(self) -> Optional[bool]:
+        # match labels to -cf labels, and if ran, return True || False
+        # if not run -cf task, return None
+        # return True = intermittent
+        # return False = regression
+        cf_to_results = collections.defaultdict(list)
+
+        for task in self.tasks:
+            for result in task.results:
+                if (
+                    task.label
+                    and task.label.endswith("-cf")
+                    and result.group == self.name
+                ):
+                    cfname = get_configuration_from_label(task.label.strip("-cf"))
+                    cf_to_results[cfname].append(result.ok)
+
+        for cfname in cf_to_results:
+            if cf_to_results[cfname]:
```

Comment:
nit: The if is unnecessary, because you are already iterating through keys in cf_to_results